src/lib/theme.scss

```diff
@@ -4,7 +4,7 @@
 @use "styles/theme-constants/colors";
 @use "styles/theme-constants/svg";
 
-:root {
+:root, :host {
```

Comment:
Do we need this @adamkudrna @mbohal ?

Reply:
Yes, we do need it. Otherwise the CSS custom properties would not be available inside shadow DOM and the live preview components would not be styled.

While rendering components inside a shadow DOM (as Docoff does it) might not be typical, there is no reason why to purposely prevent it